python/GafferUI/PlugValueWidget.py

```diff
@@ -453,6 +454,17 @@ def __plugFlagsChanged( self, plug ) :
 		if plug.isSame( self.__plug ) :
 			self._updateFromPlug()
 
+	def __plugMetadataChanged( self, nodeTypeId, plugPath, key, plug ) :
+
+		if self.__plug is None :
+			return
+
+		if (
+			Gaffer.affectedByChange( self.__plug, nodeTypeId, plugPath, plug ) or
```

Comment:
I thought that would come up again, and I can see your point. For coding, I think I actually prefer not having another namespace, at least in C++ where you have to explicitly include the file, and strong typing will tell you if what you're doing makes sense. For Python and for documentation, I can definitely see the benefit of things being namespaced into sensible sections. I think last time I was on the fence, and we compromised on "if we change this, we should do it across the board all at once".